### DISCUSSION TITLE
spatie not support onlyTrashed

### DISCUSSION THREAD

**Discussion by ArifHussainSun** [spatie/laravel-permission#2163] [Q&A]
I have a question what a really spatie role and permission  not support  onlyTrashed any Idea like this
{message: "Call to undefined method Spatie\Permission\Models\Permission::onlyTrashed()",…}

**Comment by erikn69** [comment]
You can override everything, and everything is configurable

**Reply by ArifHussainSun** [reply]
thank you so much

**Reply by erikn69** [reply]
> thank you so much

Mark this as answered
